openpower/platform/Config.in

```diff
@@ -2,3 +2,11 @@ config BR2_OPENPOWER_PLATFORM
     bool "Enable the OpenPower Platform"
     default n
     depends on BR2_powerpc64 || BR2_powerpc64le
+
+config BR2_OPENPOWER_POWER8
```

Comment:
I think you should replace BR2_OPENPOWER_PATFORM with your two different options for P9 and P8.

Alternatively, you could have BR2_OPENPOWER_POWER8 and P9 select BR2_OPENPOWER_PATFORM.